base_phone/wizard/reformat_all_phonenumbers.py

```diff
@@ -58,18 +58,16 @@ def run_reformat_all_phonenumbers(self, cr, uid, ids, context=None):
                 domain = ['|', ('active', '=', True), ('active', '=', False)]
             else:
                 domain = []
-            all_ids = obj.search(cr, uid, domain, context=context)
-            for entry in obj.read(
-                    cr, uid, all_ids, fields, context=context):
+            all_ids = obj.search(domain)
+            for entry in obj.read(all_ids, fields):
```

Comment:
Search method returns a recordset in new API.
You can use the recordsets directly in the for loop, without using the read method.